### PR TITLE
Deprecation warning for axes and colormap

### PR DESCRIPTION
**Pull request by jorisvandenbossche**
See #208

### CODE REVIEW

File: geopandas/plotting.py [hunk 1]
```diff
@@ -114,6 +114,15 @@ def plot_series(s, cmap='Set1', ax=None, linewidth=1.0, figsize=None, **color_kw
 
         matplotlib axes instance
     """
+    if 'colormap' in color_kwds:
+        warnings.warn("'colormap' is deprecated, please use 'cmap' instead "
+                      "(for consistency with matplotlib)", FutureWarning)
+        cmap = color_kwds.pop('colormap')
```
Comment (kjordahl) [inline geopandas/plotting.py]: Should we guard against the (admittedly perverse) case in which the user specifies both `cmap` and `colormap`?

Comment (jorisvandenbossche) [inline geopandas/plotting.py]: Well, I also thought about that, and in principle we could easily raise a warning or error in such a case, but I don't know that is really necessary

Comment (kjordahl) [inline geopandas/plotting.py]: OK, let's consider the deprecation warning good enough on that score.

**Comment by kjordahl** [comment]
:+1:

**Pull request merged**